code/game/objects/items/blueprints.dm

```diff
@@ -101,7 +114,38 @@
 		if(get_area_type()!=AREA_STATION)
 			return
 		edit_area()
-	updateUsrDialog()
+	if(href_list["view_blueprints"])
```

Comment:
Wanna know what happens when I href exploit this? Yeah the images will get permanently stuck for somebody.